clients/drcachesim/analyzer.h

```diff
@@ -302,6 +304,9 @@ template <typename RecordType, typename ReaderType> class analyzer_tmpl_t {
         uint64_t interval_end_timestamp, int tool_idx,
         typename analysis_tool_tmpl_t<RecordType>::interval_state_snapshot_t *&result);
 
+    uint64_t
+    get_current_microseconds();
```

Comment:
I don't understand why the wall-clock time should be needed for the dynamic schedule case. We care about the timestamp in the trace, not the wall clock time right? Schedules based on the wall clock time would be highly dependent on the host system; do we really want/need that?

Reply:
A dynamic schedule ends an input's quantum when it hits a fixed amount (say 10ms) of "simulator time" which here is the system call-clock time.  You're thinking of replaying a schedule as-traced: a fixed schedule.  But we support, and generally recommend (and require, if simulating a different # of cores) a dynamic schedule.  The timestamps in the trace are still used there for preferring which next input to take at switch points but otherwise are not controlling everything.  (We discussed this final part a while back and decided that context switch representativeness was more important than strict as-traced timestamp ordering.)

Reply:
Okay, thanks for explaining.